buildpack.md

```diff
@@ -823,6 +823,7 @@ type = "<process type>"
 command = "<command>"
 args = ["<arguments>"]
 direct = false
+default = false
```

Comment:
This appears to include another PR's content.

Reply:
:/ ...should be fixed now